Examples/Tests/radiation_reaction/test_const_B_analytical/check.py

```diff
@@ -0,0 +1,191 @@
+#! /usr/bin/env python3
+
+# This script contains few simple tests for the radiation reaction pusher
+# It initializes an electron or a positron with normalized momentum in different
+# directions, propagating in a static magnetic field (along [2/7,3/7,6/7]).
+# If the initial momentum is perpendicular to the the field, there is a very
+# simple analytical formula for the gamma factor:
+# gamma(t) =  coth(t/tc - C)
+# where tc = 1./(t0 * omega_c * omega_c)
+# and omega_c =  qB/m , t0 = (2/3) * (r_e / c)
+# and r_e = classical electron radius
+# C is chosen so that gamma(0) = initial_gamma
+# If the initial momentum is parallel to the field, it should not change.
+# This test checks these predictions with a tolerance of 5%.
+# If the script is run without a command line argument, it regenerates a new
+# inputfile according to the initial conditions listed below.
+# For detailed references see:
+# 1) M. Tamburini. PhD Thesis. University of Pisa (2011)
+#   https://etd.adm.unipi.it/t/etd-11232011-111059/
+# 2) H. Spohn Europhysics Letters 50 287 (2000)
+#   https://arxiv.org/abs/physics/9911027
+# 3) H. Spohn, Dynamics of charged particles and their radiation field
+#   (Cambridge University Press, Cambridge, 2004)
+
+import numpy as np
+import sys
+import yt
+
+#Input filename
+inputname = "inputs"
+#________________________________________
+
+#Physical constants
+c = 299792458.
+m_e = 9.1093837015e-31
+q_0 = 1.602176634e-19
+classical_electron_radius = 2.81794e-15
+reference_length = 1.0e-6
+small = 1.0e-100
```

Comment:
That's over-written before being used, isn't it? Is it on purpose?

Reply:
Oh! No... it was not intended like that. I will fix...